### DISCUSSION TITLE
Missing code

### DISCUSSION THREAD

**Discussion by qinian9584** [foxglove/community#827] [Q&A] [answered]
Excuse me,  These functions，Where is the source code located  ？

main branch ，Not found

<img width="423" alt="image" src="https://github.com/foxglove/community/assets/23311267/43bc1b0e-bd1d-4b4b-a77c-90bc24793196">

**Comment by jtbandes** [comment, marked as answer]
The code is not open-source. See: https://github.com/foxglove/studio#open-source
